2021-07-20 09:57:42,329 [7] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"8572385810245746689","FShopName":"star_shops","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:42,329 [10] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"7572385810245746689","FShopName":"energy10010","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:42,381 [9] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"6572385810245746689","FShopName":"dhgate-非真实数据-勿开启同步0828","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:42,383 [6] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"6559624990734644444","FShopName":"macabaka","BeforeDays":1},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:42,922 [10] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"6572385810245746689","FShopName":"dhgate-非真实数据-勿开启同步0828","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:42,985 [6] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"6572385810245746689","FShopName":"dhgate-非真实数据-勿开启同步0828","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:43,029 [9] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"6572385810245746689","FShopName":"dhgate-非真实数据-勿开启同步0828","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:43,113 [10] WARN  seller.dhgatemessage - SyncWarn:{"task":{"FUserId":"6572369283081568257","FShopId":"6572385810245746689","FShopName":"dhgate-非真实数据-勿开启同步0828","BeforeDays":365},"methodName":"GetMessageListLoop","errorMsg":"令牌Access Token过期或不存在"}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
2021-07-20 09:57:43,198 [6] WARN  seller.dhgatemessage - SyncWarn:{"FUserId":"6572369283081568257","FShopId":"6572385810245746689","FShopName":"dhgate-非真实数据-勿开启同步0828","BeforeDays":365}
System.Exception: 令牌Access Token过期或不存在
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<GetMessageListLoop>d__7.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 359
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   在 YQTrack.Backend.DHgateMessage.DHgateMessageTaskHelper.<>c__DisplayClass6_0.<<GetMessageList>b__1>d.MoveNext() 位置 D:\Work\YQTrack.Backend.ThridPartOrderSyncService\YQTrack.Backend.ThirdPlatform\0-Src\0.4-Service\YQTrack.Backend.DHgateMessage\DHgateMessageTaskHelper.cs:行号 332
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.RetrySyntaxAsync.<>c__DisplayClass9_0.<<RetryAsync>b__1>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Polly.Retry.RetryEngine.<ImplementationAsync>d__1`1.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Polly.Policy.<ExecuteAsync>d__100.MoveNext()
